documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1545,6 +1531,56 @@ By default, no operations are skipped.
 
 |===
 
+include::{partialsdir}/modules/all-connectors/ref-connector-configuration-database-history-properties.adoc[leveloffset=+1]
+
+////
+[id="{context}-pass-through-configuration-properties-for-database-drivers"]
+==== Pass-through configuration properties for database drivers
+
+The {prodname} connector provides for pass-through configuration of the database driver.
+Pass-through database properties begin with the prefix `database.*`.
+For example, the property `database.tinyInt1isBit=false` is passed to the JDBC URL.
+Debezium strips the prefix before it passes the property to the database driver.
+
+See the Kafka documentation for more details about link:https://kafka.apache.org/documentation.html#producerconfigs[Kafka producer configuration properties] and link:https://kafka.apache.org/documentation.html#consumerconfigs[Kafka consumer configuration properties].
+
+[id="{context}-passthrough-database-history-properties-for-configuring-producer-and-consumer-clients"]
+==== Pass-through database history properties for configuring producer and consumer clients
```

Comment:
This should be part of the 2nd partial mentioned above.

Reply:
Ditto. This section no longer appears in the rendered doc, because it's commented out. I was waiting to prove out the downstream build before I remove it entirely.